### ISSUE TITLE
Could not set up API to Code Review site for Danger Jenkins

### ISSUE THREAD

**Issue by ivan-garanin** [danger/danger#1138]
# Report

## What did you do?

`bundle exec danger --verbose`
P.S tryed to run loccally with same token and everything was good
## What did you expect to happen?

Running me Dangerfile without it crashing.

## What happened instead?

ℹ Please replace this with of what happened instead.
[33m
Could not set up API to Code Review site for Danger
[0m
For your GitHub repo, you need to expose: [33mDANGER_GITHUB_API_TOKEN[0m
You may also need: DANGER_GITHUB_HOST, DANGER_GITHUB_API_BASE_URL, DANGER_OCTOKIT_VERIFY_SSL

Found these keys in your ENV: ghprbPullId, _system_type, GIT_COMMIT, RUN_DISPLAY_URL, JENKINS_URL, ROOT_BUILD_CAUSE_GITHUBPRCAUSE, EXECUTOR_NUMBER, GITHUB_PR_TRIGGER_SENDER_AUTHOR, GITHUB_PR_NUMBER, LANG, BUILD_CAUSE, BUILD_ID, rvm_ruby_string, rvm_bin_path, RUN_CHANGES_DISPLAY_URL, rvm_version, RUBY_VERSION, GEM_HOME, ENABLE_ELASTIC, JOB_BASE_NAME, SSH_AUTH_SOCK, S_COLORS, GIT_URL, HUDSON_COOKIE, XDG_SESSION_ID, USER, GITHUB_PR_CAUSE_SKIP, GITHUB_PR_TARGET_BRANCH, rvm_pretty_print_flag, BUILD_NUMBER, WORKSPACE, PWD, HUDSON_URL, _system_version, HOME, NODE_NAME, _system_name, SSH_AGENT_PID, HUDSON_SERVER_COOKIE, GITHUB_PR_BODY, JENKINS_HOME, JOB_NAME, XDG_DATA_DIRS, GITHUB_PR_TITLE, DATABASE_URL, GITHUB_REPO_GIT_URL, GITHUB_PR_STATE, _system_arch, GEM_PATH, HUDSON_HOME, JOB_DISPLAY_URL, GITHUB_PR_COND_REF, GITHUB_PR_SHORT_DESC, GITHUB_PR_URL, RAILS_ENV, BUILD_CAUSE_GITHUBPRCAUSE, rvm_path, MAIL, BOX_INTERCOM_BACKUPS_ACCESS_TOKEN, GITHUB_REPO_SSH_URL, BUILD_URL, SHELL, JOB_URL, BUILD_DISPLAY_NAME, ROOT_BUILD_CAUSE, GITHUB_PR_SOURCE_REPO_OWNER, rvm_prefix, SHLVL, GIT_BRANCH, DANGER_GITHUB_API_TOKEN, BUILD_TAG, NODE_LABELS, LOGNAME, XDG_RUNTIME_DIR, BOX_INTERCOM_BACKUPS_CLIENT_ID, MY_RUBY_HOME, INTERCOM_ACCESS_TOKEN, GITHUB_PR_HEAD_SHA, PATH, INTERCOM_SPECS_AUTH_TOKEN, NODEJS_HOME, IRBRC, GITHUB_PR_SOURCE_BRANCH, JENKINS_SERVER_COOKIE, rvm_delete_flag, _, BUNDLER_ORIG_BUNDLE_BIN_PATH, BUNDLER_ORIG_BUNDLE_GEMFILE, BUNDLER_ORIG_BUNDLER_ORIG_MANPATH, BUNDLER_ORIG_BUNDLER_VERSION, BUNDLER_ORIG_GEM_HOME, BUNDLER_ORIG_GEM_PATH, BUNDLER_ORIG_MANPATH, BUNDLER_ORIG_PATH, BUNDLER_ORIG_RB_USER_INSTALL, BUNDLER_ORIG_RUBYLIB, BUNDLER_ORIG_RUBYOPT, BUNDLE_BIN_PATH, BUNDLE_GEMFILE, BUNDLER_VERSION, RUBYOPT, RUBYLIB, MANPATH.

Failing the build, Danger cannot run without API access.
You can see more information at https://danger.systems/guides/getting_started.html

## Your Environment

* Which CI are you running on? Jenkins
* Are you running the latest version of Danger? yes, 6.0.9
* What is your Dangerfile?
Danger worked great for months before we changed the user from human to our "bot" github account with full accesses and changed DANGER_GITHUB_API_TOKEN to new one, which was created for bot with all accesses.
P.S. Not an enterprise GitHub

**Comment by orta** [comment]
Maybe your ENV token was empty? It looks like something was there

**Comment by ivan-garanin** [comment]
@orta no, I printed it in file and it was there. Sometimes it works but then something happens and he started to complain again. Nothing changed in configs/keys. But i have got error DANGER_GITHUB_API_TOKEN again

**Comment by ivan-garanin** [comment]
it works great when I try it locally (`bundle exec danger --verbose`).

**Comment by ivan-garanin** [comment]
i just cant understand why it works several weeks in a row but then suddenly stop working for weeks without any changes

**Comment by ivan-garanin** [comment]
tried with several keys with all possible accesses.

**Comment by luissantiagodev** [comment]
Is there any update on this?
Facing the same issue
